Source/Info.plist

```diff
@@ -15,7 +15,7 @@
 	<key>CFBundlePackageType</key>
 	<string>FMWK</string>
 	<key>CFBundleShortVersionString</key>
-	<string>3.0.0</string>
+	<string>$(MARKETING_VERSION)</string>
```

Comment:
I think the `$(MARKETING_VERSION)` pattern is nice in theory but in practice it causes some obscure problems. For instance, there are Bitrise workflow steps that are able to extract Xcode project information into variables that can be used by the rest of the workflow. If we use `$(MARKETING_VERSION)` instead of the raw build number, Bitrise is not able to determine what's the actual build number and we get some funky behavior as a consequence.